src/main/java/io/confluent/idesidecar/restapi/credentials/TLSConfig.java

```diff
@@ -0,0 +1,304 @@
+package io.confluent.idesidecar.restapi.credentials;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.confluent.idesidecar.restapi.exceptions.Failure;
+import io.confluent.idesidecar.restapi.exceptions.Failure.Error;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Null;
+import jakarta.validation.constraints.Size;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+@Schema(description = "SSL configuration")
+public record TLSConfig(
+    @Schema(description = "Whether to verify the server's SSL certificate. If set to false, "
+        + "the server's SSL certificate will not be verified. Defaults to true.")
```

Comment:
Can we use the `defaultValue` attribute of the `@Schema` annotation, so the default value is captured in the OpenAPI spec?